Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -117,10 +130,18 @@ protected void RunAllTests()
 
             string testsDirectory = Path.Combine(ProductTestDataDirectory, @"Inputs\");
 
+            var inputFiles = new List<string>();
```

Comment:
>inputFiles [](start = 16, length = 10)

This change was required to run all tests in the directory, to generate a comprehensive diff (we previously stopped after first file failure).